http-api-data.cabal

```diff
@@ -1,5 +1,5 @@
 name:            http-api-data
-version:         0.3.7.2
+version:         0.3.7.3
```

Comment:
please don't touch versions. If the current version is ok with GHC-8.4.1 we'll make a revision and not an upload. (and for the future: if you touch `version:` then you have to touch `Changelog` too).

Reply:
Thanks! I'll fix this